backend/app_fastapi_v2.py

```
@@ -1,8 +1,12 @@
-from fastapi import FastAPI, HTTPException, UploadFile, File, Form
+from fastapi import FastAPI, HTTPException, UploadFile, File, Form, Depends, status, Response
+from fastapi.security import OAuth2AuthorizationCodeBearer
+from fastapi.responses import RedirectResponse
```

Ruff

`fastapi.responses.RedirectResponse` imported but unused

Remove unused import: `fastapi.responses.RedirectResponse`

(F401)